views/templates/hook/product-simulator.tpl

```diff
@@ -37,104 +58,108 @@
             window.PSSimulatorAttempts = window.attempts + 1;
             if (window.attempts > 4 )
             {
+                console.log("1");
                 clearInterval(window.PSSimulatorId);
+                checkSimulatorContent(true);
                 return true;
             }
-
             if (checkSimulatorContent()) {
+                console.log("2");
                 clearInterval(window.PSSimulatorId);
+                checkSimulatorContent(true);
                 return true;
             }
-
-            if ('{$locale|escape:'quotes'}' == 'ES') {
-                if (typeof pmtSDK == 'undefined') {
-                    return false;
-                }
-                var sdk = pmtSDK;
-            } else {
-                if (typeof pgSDK == 'undefined') {
-                    return false;
-                }
-                var sdk = pgSDK;
+            if (typeof pgSDK == 'undefined') {
+                console.log("3");
+                return false;
             }
+            var sdk = pgSDK;
 
-            if (typeof sdk != 'undefined') {
-                var price = null;
-                var quantity = null;
-                var positionSelector = '{$pagantisCSSSelector|escape:'quotes'}';
-                var priceSelector = '{$pagantisPriceSelector|escape:'quotes'}';
-                var quantitySelector = '{$pagantisQuantitySelector|escape:'quotes'}';
+            var price = null;
+            var quantity = null;
+            var positionSelector = '{$pagantisCSSSelector|escape:'quotes'}';
+            var priceSelector = '{$pagantisPriceSelector|escape:'quotes'}';
+            var quantitySelector = '{$pagantisQuantitySelector|escape:'quotes'}';
 
-                if (positionSelector === 'default') {
-                    positionSelector = '.pagantisSimulator';
-                }
+            if (positionSelector === 'default') {
+                positionSelector = '.pagantisSimulator';
+            }
 
+            if (priceSelector === 'default') {
+                priceSelector = findPriceSelector();
                 if (priceSelector === 'default') {
-                    priceSelector = findPriceSelector();
-                    if (priceSelector === 'default') {
-                        price = '{$amount|escape:'quotes'}'
-                    }
+                    price = '{$amount|escape:'quotes'}'
                 }
+            }
 
+            if (quantitySelector === 'default') {
+                quantitySelector = findQuantitySelector();
                 if (quantitySelector === 'default') {
-                    quantitySelector = findQuantitySelector();
-                    if (quantitySelector === 'default') {
-                        quantity = '1'
-                    }
+                    quantity = '1'
                 }
+            }
 
-                sdk.product_simulator = {};
-                sdk.product_simulator.id = 'product-simulator';
```

Comment:
The id must be set buy the own simulator

Reply:
you can set it if you want to know it in advance, but in this case I don use it so this line is useless